### ISSUE TITLE
jsmpp documenation

### ISSUE THREAD

**Issue by vatsalnarry2** [opentelecoms-org/jsmpp#139]
Hi,

Can you please create documentation for all classes available in jsmpp.jar
Our life will become easy, we can refer online  documentation for Classes available in jar

**Comment by pmoerenhout** [comment]
Good point. Together with the SMPP 5 additions I want to make I will do some cleanup and add the JavaDoc.
